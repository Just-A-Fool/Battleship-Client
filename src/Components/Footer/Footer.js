import React, { Component } from 'react';

class Footer extends Component {

  render() {
    return (
      <footer>
        Copyright Â© since 2020
      </footer>
    )
  }
}

export default Footer;