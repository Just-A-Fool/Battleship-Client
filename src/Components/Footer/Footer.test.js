import React from 'react';
import ReactDOM from 'react-dom';
import Footer from './Footer';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';


describe('Footer component', () => {
  it('Renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Footer />, div);
    ReactDOM.unmountComponentAtNode(div);
  })

  it('Displays the Footer component when rendered', () => {
    const wrapper = shallow(<Footer />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })


  it(`Displays copyright information`, () => {
    const wrapper = shallow(<Footer/>)
    expect(wrapper.text()).toMatch(/Copyright/)
  })
  
})