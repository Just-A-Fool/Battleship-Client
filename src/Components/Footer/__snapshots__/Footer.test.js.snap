// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component Displays the Footer component when rendered 1`] = `
<footer>
  Copyright Â© since 2020
</footer>
`;
